apps/studio/components/interfaces/Auth/Policies/PolicySelection.tsx

```diff
@@ -58,6 +72,33 @@ const PolicySelection = ({
             onClick={onViewEditor}
           />
         </div>
+        <div className="bg-surface-200 rounded-md p-4 flex items-center  gap-2 text-sm">
```

Comment:
we don't have a component for this?

Reply:
opted for alert shadcn component instead